### PR TITLE
CalculatorTest

### PR DESCRIPTION
**Pull request by Rwsmith8692**

### CODE REVIEW

File: CalculatorTests.Test/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+{
+    public class UnitTest1
+    {
+
+        [Theory]
+        [InlineData(3, 5, 8)]
+        [InlineData(6, 4, 10)]
+        [InlineData(8, 6, 14)]
+        [InlineData(7, 2, 9)]
+        public void Adding(int num1, int num2, int num3)
```
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:15]: The 3rd parameter should be `int expected` in order for your test function properly

File: CalculatorTests.Test/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        }
+
+
+         [Theory]
+        [InlineData(3, 5, -2)]
+        [InlineData(6, 4, 2)]
+        [InlineData(8, 6, 2)]
+        [InlineData(7, 2, 5)]
+
+        public void Subtracting (int num1, int num2, int num3)
```
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:29]: The last parameter should be `int expected` in order for your test function properly

File: CalculatorTests.Test/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+            Assert.Equal(num3, actual);
+        }
+
+        [Theory]
+        [InlineData(3, 5, 15)]
+        [InlineData(6, 4, 24)]
+        [InlineData(8, 6, 48)]
+        [InlineData(7, 2, 14)]
+
+        public void Multiplying (int num1, int num2, int num3)
```
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:42]: The last parameter should be `int expected` in order for your test function properly

File: CalculatorTests.Test/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+            Assert.Equal(num3, actual);
+        }
+
+        [Theory]
+        [InlineData(8, 2, 4)]
+        [InlineData(12, 4, 3)]
+        [InlineData(20, 5, 4)]
+        [InlineData(120, 20, 6)]
+
+        public void Dividing (int num1, int num2, int num3)
```
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:55]: The last parameter should be `int expected` in order for your test function properly

File: CalculatorTests.Test/UnitTest1.cs [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+            Assert.Equal(num3, actual);
+        }
+
+        [Theory]
+        [InlineData(3, 6)]
+        [InlineData(9, 368880)]
+        [InlineData(8, 40320)]
+        [InlineData(7, 5040)]
+
+        public void Factorial (int num1, int num2)
```
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:68]: The last parameter should be `int expected` in order for your test function properly

File: CalculatorTests.Test/UnitTest1.cs (hunk 2 above)
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:29]: With subtraction, order is important (e.g., 7-4 != 4-7). Which parameter is subtracted from which? "num1" and "num2" don't help us know.

File: CalculatorTests.Test/UnitTest1.cs [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+            Calculator challenger = new Calculator();
+            int actual = challenger.Multiplying(num1, num2);
+            Assert.Equal(num3, actual);
+        }
+
+        [Theory]
+        [InlineData(8, 2, 4)]
+        [InlineData(12, 4, 3)]
+        [InlineData(20, 5, 4)]
+        [InlineData(120, 20, 6)]
```
Comment (imazizahurt) [inline CalculatorTests.Test/UnitTest1.cs:53]: There's a suspiciously absent test cast, which is when dividing by zero. The tests should establish what the method being tested is supposed to return if given various inputs, including edge cases such as zero for a divisor
